homeassistant/components/spc/alarm_control_panel.py

```diff
@@ -34,16 +34,12 @@ def _get_alarm_state(area: Area) -> AlarmControlPanelState | None:
     return mode_to_state.get(area.mode)
 
 
-async def async_setup_platform(
-    hass: HomeAssistant,
-    config: ConfigType,
-    async_add_entities: AddEntitiesCallback,
-    discovery_info: DiscoveryInfoType | None = None,
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
```

Comment:
Please rebase your branch as we now have a different type for the `async_add_entities`